src/common/Navigation/styled.js

```diff
@@ -11,31 +11,27 @@ export const BlackBar = styled.div`
 
 export const NavbarContainer = styled.nav`
     width: 1368px;
-    display: flex;
-    justify-content: space-between;
+    display: grid;
+    grid-template-columns: 1fr 1fr 2.5fr;
+    padding: 23px 16px;
     align-items: center;
```

Comment:
Default value of align-items is stretch. Here are [some information about alignment in grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Box_Alignment_in_CSS_Grid_Layout). To be fair this line might be redundant in this case, but i am not entirely sure about it and cant check it right now because i am using phone and have no access to computer.